patches/v8/0025-Add-more-Error-constructor-methods.patch

```diff
@@ -0,0 +1,48 @@
+From e941ed4d066e60db0a5f7da5022ed8aa463c5d15 Mon Sep 17 00:00:00 2001
+From: James M Snell <jsnell@cloudflare.com>
+Date: Thu, 29 May 2025 15:07:37 -0700
+Subject: [PATCH] Add more Error constructor methods
```

Comment:
Can this be upstreamed?

Reply:
Maybe? I don't want to worry about up streaming yet. Let's make sure this is what we want first.